demo/src/main/java/demos/components/ToggleButtonDemo.java

```diff
@@ -30,11 +28,7 @@ public void start(Stage stage) throws Exception {
         pane.getChildren().add(toggleButton);
 
         JFXToggleNode node = new JFXToggleNode();
-        final GlyphIcon heartIcon = GlyphsBuilder.create(FontAwesomeIconView.class)
-            .glyph(FontAwesomeIcon.HEART)
-            .build();
-        heartIcon.setStyle("-fx-padding: 10");
```

Comment:
**Note:** Didn't work anyway.

---

**Before:**
![image](https://user-images.githubusercontent.com/3246162/57586453-70843480-74f6-11e9-8d56-44e181fa9a51.png)

**After:**
![image](https://user-images.githubusercontent.com/3246162/57586451-63674580-74f6-11e9-93d5-427bb2174a25.png)

Reply:
it looks better with the padding :), feel free to add it to the PR

Reply:
Changed